sdk/spring/azure-spring-boot-test-aad/src/test/java/com/azure/test/aad/selenium/AADSeleniumITHelper.java

```diff
@@ -1,115 +1,61 @@
+public class AADSeleniumITHelper extends SeleniumITHelper {
+
+    private String username;
+    private String password;
 
     static {
         DEFAULT_PROPERTIES.put("azure.activedirectory.tenant-id", AAD_TENANT_ID_1);
         DEFAULT_PROPERTIES.put("azure.activedirectory.client-id", AAD_MULTI_TENANT_CLIENT_ID);
         DEFAULT_PROPERTIES.put("azure.activedirectory.client-secret", AAD_MULTI_TENANT_CLIENT_SECRET);
         DEFAULT_PROPERTIES.put("azure.activedirectory.user-group.allowed-groups", "group1");
         DEFAULT_PROPERTIES.put("azure.activedirectory.post-logout-redirect-uri", "http://localhost:${server.port}");
-
-        final String directory = "src/test/resources/driver/";
-        final String chromedriverLinux = "chromedriver_linux64";
-        final String chromedriverWin32 = "chromedriver_win32.exe";
-        final String chromedriverMac = "chromedriver_mac64";
-        String osName = System.getProperty("os.name").toLowerCase();
-        Process process = null;
-        try {
-            File dir = new File(directory);
-            if (Pattern.matches("linux.*", osName)) {
-                process = Runtime.getRuntime().exec("chmod +x " + chromedriverLinux, null, dir);
-                process.waitFor();
-                System.setProperty(ChromeDriverService.CHROME_DRIVER_EXE_PROPERTY, directory + chromedriverLinux);
-            } else if (Pattern.matches("windows.*", osName)) {
-                System.setProperty(ChromeDriverService.CHROME_DRIVER_EXE_PROPERTY, directory + chromedriverWin32);
-            } else if (Pattern.matches("mac.*", osName)) {
-                process = Runtime.getRuntime().exec("chmod +x " + chromedriverMac, null, dir);
-                process.waitFor();
-                System.setProperty(ChromeDriverService.CHROME_DRIVER_EXE_PROPERTY, directory + chromedriverMac);
-            } else {
-                throw new IllegalStateException("Unrecognized osName. osName = " + System.getProperty("os.name"));
-            }
-        } catch (InterruptedException | IOException e) {
-            throw new RuntimeException(e);
-        } finally {
-            if (process != null) {
-                process.destroy();
-            }
-        }
     }
 
-    public AADSeleniumITHelper(Class<?> appClass, Map<String, String> properties) throws InterruptedException {
+    public AADSeleniumITHelper(Class<?> appClass, Map<String, String> properties) {
         username = AAD_USER_NAME_1;
         password = AAD_USER_PASSWORD_1;
-        app = new AppRunner(appClass);
-        DEFAULT_PROPERTIES.forEach(app::property);
-        properties.forEach(app::property);
-
-        ChromeOptions options = new ChromeOptions();
-        options.addArguments("--headless");
-        options.addArguments("--incognito", "--no-sandbox", "--disable-dev-shm-usage");
-        this.driver = new ChromeDriver(options);
-
-        this.app.start();
-        login();
+        this.appClass = appClass;
```

Comment:
If there are no further references of appclass and properties in this class, we should not keep them as fields.